base/applications/taskmgr/perfdata.c

```diff
@@ -211,12 +215,16 @@ void PerfDataRefresh(void)
 
     /* Get processor time information */
     SysProcessorTimeInfo = (PSYSTEM_PROCESSOR_PERFORMANCE_INFORMATION)HeapAlloc(GetProcessHeap(), 0, sizeof(SYSTEM_PROCESSOR_PERFORMANCE_INFORMATION) * SystemBasicInfo.NumberOfProcessors);
+    if (!SysProcessorTimeInfo)
```

Comment:
I'd prefer just skipping the work related to this (i.e. the filling of the buffer up here and the `Save system processor time info` section below) but continuing on with the rest.